Formula/o/opencsg.rb

```diff
@@ -18,17 +18,19 @@ class Opencsg < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:  "b37ea3b9c6749f57cf8b91102f334a161b707a27e9c5d4f939ea946b445207e0"
   end
 
-  depends_on "qt" => :build
+  depends_on "cmake" => :build
   depends_on "glew"
```

Comment:
No linkage -- we should check whether this is still needed. If it is, the lack of linkage needs fixed.